pkg/system-probe/config/adjust_npm.go

```diff
@@ -82,6 +83,17 @@ func adjustNetwork(cfg model.Config) {
 
 	limitMaxInt(cfg, spNS("offset_guess_threshold"), maxOffsetThreshold)
 
+	ratioPath := spNS("closed_connections_buffer_threshold_ratio")
+	if !cfg.IsConfigured(ratioPath) {
+		cfg.Set(ratioPath, defaultClosedConnectionsBufferThresholdRatioSystemProbe, model.SourceDefault)
+	} else {
+		ratio := cfg.GetFloat64(ratioPath)
+		if ratio <= 0 || ratio > 1.0 {
+			log.Warnf("Invalid value %f for %s, using default %f. Value must be > 0 and <= 1.0.", ratio, ratioPath, defaultClosedConnectionsBufferThresholdRatioSystemProbe)
+			cfg.Set(ratioPath, defaultClosedConnectionsBufferThresholdRatioSystemProbe, model.SourceAgentRuntime)
+		}
```

Comment:
suggestion: create `validateFloat64` which mirrors the other `validate*` functions in `adjust.go` and use that.

Reply:
removed the block in question